### ISSUE TITLE
Should the Updated field reflect metadata changes?

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#1454]
Presently, the Updated date on a document reflects the last time *any* metadata changed, including document tags, flags, title changes.

I use the Updated field to schedule my reading--by reading in LRU order--so this interferes somewhat with my scheduling. If you think it is worthwhile to keep this behavior, perhaps a new column could be introduced that reflects only in-document changes--pagemarks, annotations, comments. Such a 'Last Read' column would better reflect what I am interested in, and probably I'm not alone.

**Comment by thiswillbeyourgithub** [comment]
I thought your idea was strange at first but it's actually interesting to me after all.

I would too think that there is a need for a "last edited" column and a "last modified state" or something like that.

The first one would only be about the file's last internal edits (highlights, comments etc) while the second one would include change to flags, title etc.

All rows would not have to appear at the same time of course, the usual UI would work well : right clicking on the "sorting bar" at the top of the document list would allow for choosing which columns are displayed.

**Comment by burtonator** [comment]
This isn't a bug but a feature request, correct?

The field we have now is lastUpdated (AKA updated) but I think you want something like lastEdited (edited).  

Adding this to the backend is straight forward but adding it to the mutators and data manipulation might take some work.  We could get this in after 2.0 is out.

**Comment by sopoforic** [comment]
Yeah, it's a feature request--or, perhaps, a report that the current behavior is something of a misfeature.

My problem would also be solved by a reading scheduler, such as I've suggested previously, or some other feature to help manage a large reading queue (not sure what that would look like, though).
